getting-started/sessions/main.go

```diff
@@ -71,7 +83,7 @@ func newApp(projectID string) (*app, error) {
 		log.Fatalf("firestoregorilla.New: %v", err)
 	}
 
-	tmpl, err := template.New("Index").Parse("<body bgcolor={{.color}}>Views {{.views}}</body>")
+	tmpl, err := template.New("Index").Parse(`<body>{{.views}} {{if eq .views 1.0}}view{{else}}views{{end}} for "{{.greeting}}"</body>`)
```

Comment:
Are you sure you want to work with a template here instead of just posting plain text? I know that the template is much nicer and idiomatic - but is it confusing for the devs looking at this tutorial for the firs time?

Reply:
Yes. See https://github.com/GoogleCloudPlatform/golang-samples/pull/955#discussion_r317703082.